templates/20years/index.html

```diff
@@ -897,6 +1041,43 @@ <h2>Relive your best Ubuntu memories</h2>
               <p>
                 No separating the art from the artist &ndash; all credit to our amazing community! Check out the voting and submissions from our previous releases:
               </p>
+              <div class="p-section--shallow">
+                <hr class="p-rule--muted" />
+                <p>
+                  <strong>
+                    <a href="https://discourse.ubuntu.com/t/oracular-oriole-24-10-wallpaper-competition/46455">Oracular Oriole (24.10)</a>
+                  </strong>
+                </p>
+                <div class="row">
+                  <div class="col-3 col-medium-2">
```

Comment:
I wasn't sure  about the images here. Could anyone who knows provide suggestions on which images to use?